metrics/report/example.toml

```diff
@@ -0,0 +1,23 @@
+# [mail] section specifies information for
+# a SMTP server authentication as well as the
+# sender and receipts of the report email.
+# This is in order to send the obtained results
+# from checktool to an email whitelist.
+[mail]
+smtp = "myserver.smtp.com"
+port = "25"
+user = ""
+identity = ""
+pass = ""
+from = "pnp@clear-containers.com"
```

Comment:
oh yeah, I just put that fake information as an example, but its better to follow "@example.com" in order to clarify that. great I will fix that